src/Runtime/SonataPageRuntime.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\PageBundle\Runtime;
+
+use Sonata\PageBundle\Request\RequestFactory;
+use Symfony\Component\HttpKernel\HttpKernelInterface;
+use Symfony\Component\Runtime\Runner\Symfony\HttpKernelRunner;
+use Symfony\Component\Runtime\RunnerInterface;
+use Symfony\Component\Runtime\SymfonyRuntime;
+
```

Comment:
It would  be awesome if you could write test for this class.

Reply:
I don't really know how and what to test on a runtime context. as HttpKernelRunner is allready tested in symfony and createFromGlobals too.

Reply:
No idea either, that's why I won't enforce a test, (it would just be awesome ^^)

Reply:
it s a challange and it's cool,  so may be it can be another ticket just for this task  ?

Reply:
I just ask for test because it give us confidence in merge future fix or features regarding with this.
I changed the company and my current job is not using this bundle. then I do not have a real project to test it :/ .

Reply:
But in other hand I do not want to push you down because of this, Because I know that in the beginner it takes time to be familiar with the code, then I would say it is just a wish for now :)